disnake/embeds.py

```diff
@@ -457,6 +482,11 @@ def set_image(self, url: Optional[Any] = MISSING, *, file: File = MISSING) -> Se
             Passing a :class:`disnake.File` object will make the embed not
             reusable.
 
+        .. warning::
+            If used with the other ``set_*`` methods you must ensure
+            that the :attr:`.File.filename` is different than the other file(s)
+            that you are passing.
```

Comment:
```suggestion
            If used with the other ``set_*`` methods, you must ensure
            that the :attr:`.File.filename` is unique to avoid duplication.
```